inference-engine/tests/functional/plugin/cpu/subgraph_tests/include/conv_concat.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+
+#include "../../test_utils/cpu_test_utils.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace CPUTestUtils;
+
+namespace LayerTestsDefinitions {
+
+enum class nodeType {
+    convolution,
+    convolutionBackpropData,
+    groupConvolution,
+    groupConvolutionBackpropData
+};
+
+std::string nodeType2PluginType(nodeType nt) {
+    if (nt == nodeType::convolution) return "Convolution";
+    if (nt == nodeType::convolutionBackpropData) return "Deconvolution";
+    if (nt == nodeType::groupConvolution) return "Convolution";
+    if (nt == nodeType::groupConvolutionBackpropData) return "Deconvolution";
+    assert(!"unknown node type");
+    return "undef";
+}
+
+std::string nodeType2str(nodeType nt) {
+    if (nt == nodeType::convolution) return "Convolution";
+    if (nt == nodeType::convolutionBackpropData) return "ConvolutionBackpropData";
+    if (nt == nodeType::groupConvolution) return "GroupConvolution";
+    if (nt == nodeType::groupConvolutionBackpropData) return "GroupConvolutionBackpropData";
+    assert(!"unknown node type");
+    return "undef";
+}
```

Comment:
Looks like nodeType2PluginType and nodeType2str are identical. Lets remove one.

Reply:
```nodeType2PluginType``` using to convert ngraph node type to cpu plugin node type and using in ```CheckCPUImpl```.
```nodeType2str``` using in ```getTestCaseName``` and print ngraph node name.
So I believe we should stay both functions